ansible/vars/softlayer/softlayer_public.yml

```diff
@@ -270,3 +270,5 @@ localsettings:
   USE_PARTITIONED_DATABASE: True
   WS4REDIS_CONNECTION_HOST: "{{ groups.redis.0 }}"
   XFORMS_PLAYER_URL: "http://{{ groups.touchforms.0 }}:4444/"
+  ENIKSHAY_PRIVATE_API_USERS: {'BI': 'ppia-bi21', 'GU': 'ppia-gu21', 'MH': 'ppia-mh14'}
```

Comment:
there is a similar declaration in the same file.
https://github.com/dimagi/commcarehq-ansible/pull/779/files#diff-55d98789f0f1ed501c2fc3bf8a9a0c9bR176